### DISCUSSION TITLE
通过 iStoreOS 中 Docker 安装的 1Panel，防火墙服务问题

### DISCUSSION THREAD

**Discussion by IJayHuI** [1Panel-dev/1Panel#6692] [Q&A]
IStoreOS基于openWrt，openWrt使用opkg作为包管理器。查看官网的教程是使用yum或者apt安装firewalld或者ufw。不知道openWrt这种怎么处理。
![屏幕截图 2024-10-12 224952](https://github.com/user-attachments/assets/2a0cc600-e795-4259-a331-bb78ce12a2cb)

**Comment by wanghe-fit2cloud** [comment]
参考一下这个 issue，看是否有帮助。
https://github.com/1Panel-dev/1Panel/issues/3524

**Reply by IJayHuI** [reply]
这个 issue 的时间，istoreos他们团队还没有做这个插件，这个插件好像是7月份左右出来的。这里面我看也完全没有提到关于防火墙的问题。

**Reply by wanghe-fit2cloud** [reply]
1Panel 官方目前尚未适配 OpenWrt。如果你必须使用 ufw 或 firewalld，可能需要手动进行源码编译和安装。但这种方式并不推荐，因为手动编译安装复杂且可能带来额外问题。

**Reply by IJayHuI** [reply]
好的，了解
